openvswitch.spec

```diff
@@ -117,7 +117,7 @@ Requires: openssl iproute module-init-tools
 
 Requires(post): /usr/bin/getent
 Requires(post): /usr/sbin/useradd
-Requires(post): /usr/bin/sed
+Requires(post): sed
```

Comment:
Linked bz resolved this with:
-Requires(post): /usr/bin/sed
+Requires(post): /bin/sed
which works in both Fedora and RHEL.